packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -43,13 +43,17 @@ export function buildRootObject(powers, _params, testJigSetter = undefined) {
     const issuerTable = makeIssuerTable();
     const getAmountMath = brand => issuerTable.getByBrand(brand).amountMath;
 
-    const invitationHandleToHandler = makeWeakStore('invitationHandle');
+    /** @type {WeakStore<InvitationHandle,OfferHandler>} */
+    const invitationHandleToHandler = makeWeakStore('InvitationHandle');
 
     /** @type {Store<ZCFSeat,ZCFSeatAdmin>} */
     const zcfSeatToZCFSeatAdmin = makeStore('zcfSeat');
     /** @type {WeakStore<ZCFSeat,SeatHandle>} */
     const zcfSeatToSeatHandle = makeWeakStore('zcfSeat');
 
+    /** @type {WeakStore<InvitationHandle, ZcfSeatPack>} */
```

Comment:
Why is this type named `ZcfSeatPack` instead of `ZcfSeatKit`?

Reply:
There is already a ZcfSeatKit, which  is a different thing. `makeEmptySeatKit()` returns a zcfSeat and a UserSeat, and this is typed as ZcfSeatKit. Since that's externally visible, I didn't want to rename it to something more appropriate. This new object has a stronger claim to the name.

I should rename the collection to invitationHandleToZcfSeatPack unless we are happy with a term rotation.

Reply:
```suggestion
    /** @type {WeakStore<InvitationHandle, ZcfSeatAdminKit>} */
```

It seems ZcfSeatPack 
https://github.com/Agoric/agoric-sdk/blob/a15da1cd3e594c313eaf94112b7cbe899736dcf2/packages/zoe/src/internal-types.js#L119-L126
is the same as ZcfSeatAdminKit
https://github.com/Agoric/agoric-sdk/blob/a15da1cd3e594c313eaf94112b7cbe899736dcf2/packages/zoe/src/internal-types.js#L67
so please rename all uses of ZcfSeatPack to ZcfSeatAdminKit

Reply:
@michaelfig I'm curious: Are the two notations above for defining a record type in jsdoc equivalent? Which should we prefer?

Reply:
The second is for concision when the properties are obvious.  The first allows you to add comments to the different properties.